cache-config/t3c-diff/README.md

```diff
@@ -58,12 +58,37 @@ If one file exists but the other doesn't, it will always be a diff.
 Note this means there may be no diff text printed to stdout but still exit 1 indicating a diff
 if the file being created or deleted is semantically empty.
 
+Mode is file permissions in octal format, default is 0644.
+Line comment is a character that signals the line is a comment, default is #
+
+Uid is the user id of the file's owner, default is running process's uid.
+Gid is the id of the group the file is a member of, default  running process's gid.`
+
 # OPTIONS
 
+-a, -\-file-a
+
+    Path to first diff file, can also be stdin.
+
+-b, -\-file-b
+    Path to second diff file, can also be stdin.
+
+-g, -\-file-gid
+    Group id the file being checked should have.
```

Comment:
This is a fine description - why not put it in the help text?

Reply:
Updated help text